fhir-server-test/src/test/java/org/linuxforhealth/fhir/server/test/FHIRServerTestBase.java

```diff
@@ -567,23 +568,13 @@ protected boolean isTransactionSupported() throws Exception {
                 assertEquals(RestfulCapabilityMode.SERVER.getValue(), rest.getMode().getValue());
                 if (rest.getInteraction() != null) {
                     for (org.linuxforhealth.fhir.model.resource.CapabilityStatement.Rest.Interaction interaction : rest.getInteraction()) {
-                        if (interaction.getCode().getValue().equals(SystemRestfulInteraction.TRANSACTION.getValue())) {
+                        if (interaction.getCode().getValueAsEnum() == SystemRestfulInteraction.TRANSACTION.getValueAsEnum()) {
```

Comment:
FWIW I was trying to recommend 
`interaction.getCode().getValueAsEnum() == SystemRestfulInteraction.Value.TRANSACTION` but I think this is fine